odoo/netsvc.py

```diff
@@ -70,11 +75,25 @@ def emit(self, record):
 }
 
 class DBFormatter(logging.Formatter):
+    def formatException(self, exc_info):
+        result = super(DBFormatter, self).formatException(exc_info)
+        # Also provide the error's class & module name
+        return '<<<< ' + exc_info[0].__module__ + '.' + exc_info[0].__name__ + ' >>>>\n' + result
```

Comment:
Maybe this it to much of a braking change, but I wanted to hear your opinion.
I think ti will help reading the log stream during development...